### DISCUSSION TITLE
Run in WebIDE

### DISCUSSION THREAD

**Discussion by schealex** [UI5/webcomponents-react#1355] [Q&A]
Hey guys,

is there a way to run a app that was build with the webcomponents based on create-react-app in the webide?
i got the MTA building working but i'm unable to start the app in the webide.

any help on this is greatly appreiated

regards
Alex

**Comment by MarcusNotheis** [comment]
Hey @schealex,

if I have understood you correctly the "build and deploy" part is already working for you but you want to start the "localhost" of `react-scripts` inside web ide?

**Comment by schealex** [comment]
Hey @MarcusNotheis thanks for the response. Building and deploying i can do locally (using NPM) but not from the WebIDE.
I'm trying to get the project started from the WebIde.

**Comment by MarcusNotheis** [comment]
I've just tried to setup a project with MTA in the WebIDE but I couldn't get it running as well - but I have to admit that I usually never use the WebIDE so unfortunately I can't offer any valuable help here.
I would transfer this issue now into an discussion, maybe someone else has ideas 🙂
